src/deluge/gui/views/timeline_view.h

```diff
@@ -70,4 +70,8 @@ class TimelineView : public RootUI {
 
 	// ui
 	UIType getUIType() { return UIType::TIMELINE_VIEW; }
+
+private:
+	uint32_t timeHorizontalZoomLastHitClipMax = 0;
+	int8_t delayedHorizontalScrollDirection;
```

Comment:
this can only be -1 or +1 right? would be good to document that (and the meaning of the variable above as well)

Reply:
How's what I adeed?

Reply:
left a suggestion so doxygen will pick it up, but otherwise looks fine. thanks!